pygmt/src/rose.py

```diff
@@ -0,0 +1,215 @@
+"""
+rose - Plot windrose diagrams or polar histograms.
+"""
+
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    A="sector",
+    B="frame",
+    C="cmap",
+    D="shift",
+    Em="vectors",
+    F="no_scale",
+    G="color",
+    I="inquire",
+    JX="diameter",
+    L="labels",
+    M="vector_params",
+    Q="alpha",
+    R="region",
+    S="norm",
+    T="orientation",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    Z="scale",
+    i="columns",
+    c="panel",
+    p="perspective",
+    t="transparency",
+)
+@kwargs_to_strings(R="sequence", i="sequence_comma", p="sequence")
```

Comment:
```suggestion
@kwargs_to_strings(R="sequence", c="sequence_comma", i="sequence_comma", p="sequence")
```

Reply:
Will leave it as is for the moment (see @weiji14 's comment below).

Reply:
Actually this just allows panel (c) to be a list so is fine to commit.